sqlglot/parser.py

```diff
@@ -1410,20 +1410,18 @@ def _parse_property(self) -> t.Optional[exp.Expression]:
         if self._match_text_seq("SQL", "SECURITY"):
             return self.expression(exp.SqlSecurityProperty, definer=self._match_text_seq("DEFINER"))
 
-        assignment = self._match_pair(
-            TokenType.VAR, TokenType.EQ, advance=False
-        ) or self._match_pair(TokenType.STRING, TokenType.EQ, advance=False)
+        index = self._index
+        key = self._parse_column()
 
-        if assignment:
-            key = self._parse_var_or_string()
```

Comment:
This PR is now breaking on an AST level because `key`s have become `Identifier`s (current) vs `Var`s (before). Besides that I think it should be fine?